pages/index.js

```diff
@@ -1,4 +1,13 @@
 const html = require('html-template-tag')
+const homepageData = require('../public/data/homepage.json')
+const md = require('markdown-it')()
+
+const Blurb = ({ title, body }) => html`
+  <div class="Blurb">
+    <h2 class="Blurb-title">${title}</h2>
+    $${md.render(body)}
```

Comment:
Does this do escaping as well?

Reply:
Yebo yes

`$${whater}` ading the extra `$` sign before the interpolated variable unescapes the HTML